verification/abi.go

```diff
@@ -84,17 +84,17 @@ const (
 	CertifyKeyCsr  CertifyKeyFormat = 1
 )
 
-type CertifyKeyReq[Digest DigestAlgorithm] struct {
+type CertifyKeyReq struct {
 	ContextHandle ContextHandle
 	Flags         CertifyKeyFlags
-	Label         Digest
+	Label         []byte
 	Format        CertifyKeyFormat
 }
 
-type CertifyKeyResp[CurveParameter Curve, Digest DigestAlgorithm] struct {
+type CertifyKeyResp struct {
 	NewContextHandle  ContextHandle
-	DerivedPublicKeyX CurveParameter
-	DerivedPublicKeyY CurveParameter
+	DerivedPublicKeyX []byte
```

Comment:
Internally each client implementation passes fixed-size structs which use generics to `execCommand`. It copies the values to/from these structs (w/o generics).

I want the tests to care as little as possible about the profile. This change detects the profile of the target and then returns the right client for that profile.

Reply:
The follow up to this will add a `NewClient` API to TestDPEInstance which returns a `*Client` (which doesn't work with generics if the caller doesn't know what profile the target is going to be using).

Reply:
I think that's reasonable. Also means fewer anonymous structs. I'll make that change.

Reply:
I had a better idea: https://github.com/chipsalliance/caliptra-dpe/pull/220